### ISSUE TITLE
TLS version for azurerm_windows_function_app

### ISSUE THREAD

**Issue by sreekanth3107** [hashicorp/terraform-provider-azurerm#27842]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave comments along the lines of "+1", "me too" or "any updates", they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Description

Microsoft announced that they are supporting App service TLS versions 1.3, but Terraform is not yet supported for 1.3.
https://learn.microsoft.com/en-us/azure/app-service/overview-tls

### New or Affected Resource(s)/Data Source(s)

azurerm_windows_function_app

### Potential Terraform Configuration

```hcl
resource "azurerm_windows_function_app" "example" {
  name                = "example-windows-function-app"
  resource_group_name = azurerm_resource_group.example.name
  location            = azurerm_resource_group.example.location

  storage_account_name       = azurerm_storage_account.example.name
  storage_account_access_key = azurerm_storage_account.example.primary_access_key
  service_plan_id            = azurerm_service_plan.example.id

  site_config {
minimum_tls_version = 1.3
}
}
```

**Comment by ziyeqf** [comment]
Hi @sreekanth3107, thanks for opening the issue.

May I confirm which AzureRM provider version did you use? the TLS 1.3 support should have been done on https://github.com/hashicorp/terraform-provider-azurerm/pull/27196 and released on v4.3.0, do you mind upgrading you AzureRM provider and try again?

**Comment by rcskosir** [comment]
The documentation was updated via PR https://github.com/hashicorp/terraform-provider-azurerm/pull/27823 in 4.8.0, as such, I am going to mark this issue as closed, if that's not the case please provide more information regarding the issue you are facing, thank you!
